examples/others/hexaBeam/hexaBeam_gravity.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+import os
+import Sofa
+
+plugins=["SofaPython3","SoftRobots","ModelOrderReduction","STLIB",
+        
+         "Sofa.Component.Visual",
+         "Sofa.Component.AnimationLoop",
+         "Sofa.GL.Component.Rendering3D",
+         "Sofa.Component.Constraint.Lagrangian.Solver",
+         'Sofa.Component.IO.Mesh',
+         'Sofa.Component.Playback',
+         'Sofa.Component.Constraint.Lagrangian.Correction', # Needed to use components [GenericConstraintCorrection]
+         'Sofa.Component.Engine.Select', # Needed to use components [BoxROI]
+         'Sofa.Component.LinearSolver.Direct', # Needed to use components [SparseLDLSolver]
+         'Sofa.Component.Mapping.Linear', # Needed to use components [BarycentricMapping]
+         'Sofa.Component.Mass', # Needed to use components [UniformMass]
+         'Sofa.Component.ODESolver.Backward', # Needed to use components [EulerImplicitSolver]
+         'Sofa.Component.SolidMechanics.FEM.Elastic', # Needed to use components [TetrahedronFEMForceField]
+         'Sofa.Component.SolidMechanics.Spring', # Needed to use components [RestShapeSpringsForceField]
+         'Sofa.Component.StateContainer', # Needed to use components [MechanicalObject]
+         'Sofa.Component.Topology.Container.Dynamic', # Needed to use components [TetrahedronSetTopologyContainer]
+         'Sofa.Component.Constraint.Projective', # Needed to use components [FixedProjectiveConstraint]
+         'Sofa.Component.Topology.Container.Grid'] # Needed to use components [RegularGridTopology]
+        
+
+
+def createScene(rootNode):
+
+    rootNode.findData('dt').value=0.02
+    rootNode.findData('gravity').value=[0, -9810, 0]
+    rootNode.addObject('VisualStyle', displayFlags='showBehaviorModels showForceFields')
+    rootNode.addObject('RequiredPlugin', pluginName=plugins, printLog=False)
+    rootNode.addObject('FreeMotionAnimationLoop')
+    rootNode.addObject('GenericConstraintSolver', tolerance="1e-12", maxIterations="10000")
+
+
+    M1 = rootNode.addChild("M1")
+    M1.addObject('EulerImplicitSolver',rayleighStiffness="0.1", rayleighMass="0.1")
```

Comment:
```suggestion
    M1.addObject('EulerImplicitSolver', rayleighStiffness=0.1, rayleighMass=0.1)
```
I think it is preferable not to define Data with strings. This changes must be applied in the whole file (even in your other PRs)